packages/pangea-sdk/PangeaCyber.Net/Sanitize/Requests/SanitizeRequest.cs

```diff
@@ -41,18 +41,21 @@ public class SanitizeRequest : BaseRequest
         [JsonProperty("uploaded_file_name")]
         public string? UploadedFileName { get; set; }
 
-        ///
+        /// <inheritdoc cref="BaseRequest.TransferMethod" />
         [JsonIgnore]
-        public TransferMethod? RequestTransferMethod
+        public TransferMethod RequestTransferMethod
         {
-            get { return TransferMethod; }
+            get { return TransferMethod ?? Net.TransferMethod.PostURL; }
```

Comment:
Hmm, I don't like this very much. I think we should return the real value and fix setter to not allow null assignment, setting it by default to `Net.TransferMethod.PostURL` instead

Reply:
But where would we ever want this to return `null`? Also the default setting is in the constructor.

Reply:
It won't return `null` because we'll add the check in the setter to validate that we are not setting to null.
But, thinking twice, we should allow `null` value here and add the check on BaseClient, that way all those requests that has `null` transfer_method will be defaulted to `post-url`. It include file-scan and share uploads

Reply:
The thing is that `BaseRequest.TransferMethod` is null by default, and _all_ request bodies inherit that. And that's intentional (I hope) because this translates to `"transfer_method"` being omitted from most request bodies, which is correct. The differentiating factor for requests that _do_ use `"transfer_method"` is that they either:

1. Wrap it in a `RequestTransferMethod` property (this is found in Sanitize and Share).
2. Use a builder to default it to `TransferMethod.PostURL` (this is found in File Scan).

This PR is basically the second solution, just implemented via the constructor rather than a whole builder. `BaseClient` shouldn't be deciding the default because it doesn't know which requests need a `"transfer_method"` and which do not.

> It won't return null because we'll add the check in the setter to validate that we are not setting to null.

This bug isn't caused by anyone setting anything to `null`. Rather, it's caused by the default being `null`. Changing the setter will not help.

Reply:
Okey, if file-scan resolve it in the builder let's do the same here then. I'd add the check in BaseClient anyway to avoid possible bugs

Reply:
> I'd add the check in BaseClient anyway to avoid possible bugs

What would that check look like?